app/kumactl/cmd/install/testdata/install-control-plane.dump-values.yaml

```diff
@@ -138,6 +138,52 @@ controlPlane:
     path: /
     # -- Each path in an Ingress is required to have a corresponding path type. (ImplementationSpecific/Exact/Prefix)
     pathType: ImplementationSpecific
+  
+  diagnostics:
+    service:
+      # -- Port on which Diagnostics Service is exposed on Node for service of type NodePort
+      nodePort: 30680
+      # -- Port on which Diagnostics Service is exposed
+      port: 5680
+
+  apiServer:
+    http:
+      service:
+        # -- Port on which Http api server Service is exposed on Node for service of type NodePort
+        nodePort: 30681
```

Comment:
According to the [documentation](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport),
```yml
#By default and for convenience, the Kubernetes control plane will allocate a port from a range (default: 30000-32767)
``` 
A validation could be added to the deployments/charts/kuma/templates/_helpers.tpl

Reply:
I think in general any nodeport is possible so I don't the chart can/should validate it.